index.bs

```diff
@@ -514,6 +514,19 @@ See also:
 
 This section details design principles for features which are exposed via CSS.
 
+<h3 id="css-content-should-be-visible">Content should be readable by default</h3>
```

Comment:
```suggestion
<h3 id="css-content-should-be-visible">Content should be perceivable by default</h3>
```

Maybe? Using WCAG language...

Reply:
Despite 'perceivable' it's not a term used by CSS and proved to be somewhat unfamiliar for non-native speakers. The preferred title ended up being: "Content should be viewable and accessible by default".

Reply:
I can live with that.